src/uiStrings.ts

```diff
@@ -127,4 +127,6 @@ export class UiStrings {
     static readonly DatplaneAlreadyAdded = vscode.l10n.t("This Data Plane Runtime URL already added to Data View.");
     static readonly SelectItemFromTreeView = vscode.l10n.t("Select from which tree view");
     static readonly GetTreeView = vscode.l10n.t("Please connect to Azure API Center Service first.");
+    static readonly APIControlPlaneView = vscode.l10n.t("API Center Management View");

```

Comment:
"API Center Management Plane View" to algin with "API Center Data Plane View"?

Reply:
Is it shown in picker of Command Palette? Do we really need to add 'View' in string?

Reply:
change to API Center Management Plane and API Center Data Plane